src/index.ts

```diff
@@ -1,3 +1,6 @@
+/// <reference path="../typings/ethjs-util.d.ts"/>
+import * as ethjsUtil from 'ethjs-util'
```

Comment:
For unknown TS reasons, couldn't get `export * from 'ethjs-util'` to work unless it was imported at the top of the file with this path hint.

Reply:
When you solve `tsconfig.json` typo, it is not needed.